WordPress/wordpress-playground
# Add iconv to the kitchen-sink bundle

## What is this PR doing?

Includes the `iconv` PHP extension in the kitchen-sink extension bundle, see https://github.com/WordPress/wordpress-playground/pull/681

This work was done by @seanmorris in Pull Request #639, I'm only extracting it to a separate PR to ship it faster.

<img width="961" alt="CleanShot 2023-10-12 at 11 13 59@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/a275ec51-e0fe-4e3d-8e66-b36987b2a57c">

## Test plan

1. In Playground, go to `/phpinfo.php`
2. Confirm the iconv extension is not loaded by default
3. Go to the customization modal, check "load additional PHP extensions"
4. Confirm the iconv extension is now listed on `/phpinfo.php`
